docs/README.md

```diff
@@ -36,7 +37,7 @@ Same as `.table-hover` class for adding a hover effect (grey background color) o
 ### <a name='condensed'>condensed - [Bool]</a>
 Same as `.table-condensed` class for makeing a table more compact by cutting cell padding in half
 
-### <a name='cellEdit'>cellEdit - [Bool]</a>
+### <a name='cellEdit'>cellEdit - [Object]</a>
```

Comment:
typo

Reply:
lol  I already fix it on my PR...

Reply:
I think it might be alright and show no diff if we do the same thing.